qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -35,7 +35,7 @@
 #   cd $CEPH_SRC_PATH
 #   PATH=$CEPH_SRC_PATH:$PATH
 #   RBD_MIRROR_NOCLEANUP=1 RBD_MIRROR_TEMDIR=/tmp/tmp.rbd_mirror \
-#     ../qa/workunits/rbd/rbd_mirror_journal.sh
+#     ../qa/workunits/rbd/rbd_mirror_helpers.sh
```

Comment:
I think you have to use `rbd_mirror.sh journal` here, so it would actually setup the environment and run the tests.

Another approach I used to use was just to to setup the environment without actually running the tests. Then it would be:
```
./qa/workunits/rbd/rbd_mirror_helpers.sh setup
```

Reply:
Thanks @trociny addressed both points.